src/StringToVectorConverter.cpp

```diff
@@ -0,0 +1,45 @@
+#include "StringToVectorConverter.hpp"
+
+int lambda(char c){
```

Comment:
Please move this lamda inside ConvertStringToVector, because it is use only there and make it a real lambda function and use it like this:

```
    std::transform(playerFrames.begin(),
                   playerFrames.end(),
                   std::back_inserter(vec),
                   lambda);
```